pkg/helm/helm.go

```diff
@@ -325,20 +381,20 @@ func (h *Helm) LoadChart(ref *LoadChartConfig) (c *chart.Chart, err error) {
 				SkipUpdate: true,
 				Getters:    h.getters,
 			}
+
 			if err = man.Update(); err != nil {
 				return
 			}
 
-			// Update all dependencies which are present in /charts.
-			c, err = chartutil.Load(chartPath)
+			c, err = loadChartWithLockfile(chartPath, ref.LockFile)
 			if err != nil {
 				return
 			}
 		}
-	} else if e != chartutil.ErrRequirementsNotFound {
-		return nil, fmt.Errorf("cannot load requirements: %v", e)
 	}
 
+	// Check chart requirements to make sure all dependencies are present in /charts
+	c, err = chartutil.Load(chartPath)
```

Comment:
I think GitHub is having an issue. I've removed these lines from my branch but the new commit is not showing up here. Let's see if this resolves itself in a little bit.